docs/qs-backup-setup.md

```diff
@@ -82,81 +89,155 @@ You should obtain a layout similar to this one (it is fine if you only have one
+8. <a href="/content/data/sm-main-commit.bat" target="_blank" rel="noopener">Download this .bat file</a> and save it in your local repository folder (where your `.git` directory is located). It contains the following commands:
 
 ```bat
 git add -A && git commit -m "Update"
 git push
 ```
 
-7. Run `sm-main-commit.bat` (double click). If all went well, your should be able to see your collection in your GitHub repository web page.
+9. Run `sm-main-commit.bat` (double click). If all went well, your should be able to see your collection in your GitHub repository web page.
 
 #### Pushing (*\"saving\"*) your work to GitHub
 
 !> Every time you finish using SuperMemo, make sure to run `sm-main-commit.bat`.
 
 That's all ! Your collection is synchronized online, congratulations !
 
+
+
+#### Optional: Pin commit script to taskbar
+
+To make running sm-main-commit.bat more convenient, you can add it to taskbar. Windows doesn't let you add .bat files to taskbar so instead you can do the following:
+
+1. Right click on sm-main-commit.bat and click [create shortcut ![](/content/images/backup-setup/shortcut-creation.png)](/content/images/backup-setup/shortcut-creation.png '@tooltip-preview').
+
+2. The new shortcut still can't be added to taskbar. As such, right click properties and [go to shortcuts menu ![](/content/images/backup-setup/shortcut-edit.png)](/content/images/backup-setup/shortcut-edit.png '@tooltip-preview'). Add ```cmd /c``` (```cmd \c``` will not work) to the front of [ the target script ![](/content/images/backup-setup/shortcut-target.png)](/content/images/backup-setup/shortcut-target.png '@tooltip-preview') and press apply:
+
+3. You can now [right click the shortcut ![](/content/images/backup-setup/shortcut-taskbar.png)](/content/images/backup-setup/shortcut-taskbar.png '@tooltip-preview') and add it to taskbar. After you are done using SuperMemo, click the shortcut in taskbar (or use winkey + taskbar position number key) to sync changes to your collections's Github repository.
+
+#### Optional: Sync SuperMemo between computers
+
+!>As a slight mishap could cause data loss, don't try this without understanding the git commands involved. To learn more about how git works, check out [this guide](https://hackernoon.com/understanding-git-fcffd87c15a3) or this [interactive tutorial](https://learngitbranching.js.org/)
+
+The starting computer will be referred to as 'A' and the target computer you want to sync your collection to will be 'B'
+
+1. On computer A,  <a href="/content/data/sm-main-pull.bat" target="_blank" rel="noopener">download this .bat file</a> and save it in your local repository folder (where your `.git` directory is located). It contains the following commands:
+
+```bat
+git fetch --all
+git reset --hard origin/master
+```
+?>These commands override the local repository's files for the ones that you've most recently uploaded to git
```

Comment:
That's very dangerous. If you have uncommitted work, you will lose all of it after running `git reset --hard origin/master`. I'm not sure that we want to advertise such a method.

Reply:
Is there a better way to write the script that would be safer? I was thinking that if there are local changes you could have it automatically commit to a backup branch and after that overwrite with origin/master

Reply:
I think this calls for a user decision. You could add a prompt dialog:

In Batch: https://stackoverflow.com/questions/50763301/how-to-add-a-yes-no-response-in-a-batch-file
In Powershell: https://stackoverflow.com/questions/24649019/how-to-use-confirm-in-powershell/24649481

Reply:
To be honest, I don't feel very comfortable with this for now. I think it is one thing to suggest that users can synchronize their files across computers. But it is another one to provide an official method.

With the former, the user is responsible for making sure that his method doesn't endanger his collection. With the latter, we can be blamed if something goes wrong.

If we do this, it should be carefully planned. Or we can provide some guidance, with a Caveat Emptor in big, red letters with warning signs on both sides, but I'm not a fan of it.

I believe yours is the best choice though, since users will try to synchronize their collection with or without our help anyway. So we might as well get them started, and avoid them some pitfalls if we can.